doc/manual/checker_library.md

```diff
@@ -85,3 +85,52 @@ traceability). The metadata produced by a check shall be included and
 described in its documentation, (e.g., if a check logs in metadata version and
 configuration of a simulator, the checker documentation shall describe the
 content of aforementioned metadata fields)._
+
+## Manifest File
+
+Each Checker Library needs to provide a manifest file with configuration
+information for the framework. The format of this file is aligned with the one
+for the [Result File (`*.xqar`)](file_formats.md#result-file-xqar) and follows
+the same specification. The only difference is that it may contain additional
+Report Modules, which are delivered as part of the Checker Library.
+
+### Example
+
+```xml
+<CheckerResults version="1.0.0">
+    <CheckerBundle name="CheckerBundle1" description="Description of the Checker Bundle" build_date="2024-03-26" version="1.0.0">
+        <Executable name="/usr/bin/python3"> <!-- apply to Linux only! Probably just "python"? -->
```

Comment:
> "My windows" 😆 resolves `python3` to the Microsoft App Store, which I cannot connect to 🙈 . `python` resolves to a Python 3.x version of the `python.exe`.
> 
> Anyway, I change `name` --> `program`

Yeah thats a common "issue"... You can change that annoying gehavior via "Manage app execution aliases": https://stackoverflow.com/questions/58754860/cmd-opens-windows-store-when-i-type-python

Reply:
> Manifest are inherently installation specific, and I would go for a simplification. While installing a checker bundle, the installation procedure can put the correct location for the executable program in the manifest itself (e.g., starting from a manifest template). This is a developer specific implementation. In general, this will simplify installation of checker bundle that runs in virtualenv.
> 
> Some notable examples:
> 
> * for python packages installed with `pip`, a `CusomInstall.run` command can be included in `setup.py` files to generate the manifest file. System location for manifest file may be read from env variable (the same one used by the framework to scan for manifest files)
> * for linux `apt` based project, manifest may be generated in `postinst` phases
> * for windows installers based on Inno Setup, it is a file created with a custom Pascal function (yikes!)
> 
> Under this condition, we should only specify the final form of the manifest, which in general can be OS agnostic.
> 
> Additionally, I would go with the concept of `<Interpreter>` for languages that require an interpreter instead of being execute directly.

Yeah I think thats right. This will depend on installation or a development configuration. 
From the frameworks perspective, we shouldn't consider how this is filled, only that this will be somehow defined (installation wise or by the users definition).

Have you any opinion on maybe changing the control flow from qc-framework searches for manifests to checker bundles register at qc-framework? (e.g. via EnvVar).
From my point of view this would only have some issues if qc-framwork is later installed than the checker bundles. (even though every checker could have some code like 
```
if ENV_VAR not present
    ENV_VAR <= "/some/user/accessible_directory/"
fi
place manifest in ENV_VAR folder
```
But this will rely on the point, that each checker will do this stuff.